openedx/core/djangoapps/discussions/serializers.py

```diff
@@ -154,12 +154,14 @@ class DiscussionsConfigurationSerializer(serializers.ModelSerializer):
     """
     Serialize configuration responses
     """
+    user_permissions = serializers.SerializerMethodField()
 
     class Meta:
         model = DiscussionsConfiguration
         fields = [
             'enabled',
             'provider_type',
+            'user_permissions'
```

Comment:
I don't think it makes sense to add this field. We only have a single permission right now, so let's keep it simple.